src/main/java/org/testng/junit/JUnitMethodFinder.java

```diff
@@ -26,45 +24,19 @@
  *
  */
 public class JUnitMethodFinder implements ITestMethodFinder {
-  private String m_testName = null;
-  private IAnnotationFinder m_annotationFinder = null;
+  private final String m_testName;
+  private IAnnotationFinder m_annotationFinder;
```

Comment:
`final` should be possible here

Reply:
It's strange that the class is not used. I supposed it was used for the junit4 support.
I have not the history of the feature but maybe the junit support is now delegated to the junit engine directly.

@cbeust Do you remember something about this class?

Reply:
Nope :)